src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -1076,4 +1077,41 @@ class LicenceController(private val licenceService: LicenceService) {
   ) {
     licenceService.updateSentenceDates(licenceId, request)
   }
+
+  @GetMapping(value = ["/unapproved-by-crd"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @ResponseBody
+  @Operation(
+    summary = "Get details of edited licences which have not been reapproved by CRD date",
+    description = "Returns prisoner and com details " +
+      "Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Licence(s) found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = UnapprovedLicence::class))],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorised, requires a valid Oauth2 token",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden, requires an appropriate role",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Resource not found",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
```

Comment:
I think 404 would never happen? you'd just return a 200 and an empty list?

Reply:
404 removed